README.md

```diff
@@ -30,16 +34,17 @@ create user takwimu with password takwimu;
 create database takwimu;
 grant all privileges on database takwimu to takwimu;
 ```
-Run migrations to keep Django happy:
-```
-python manage.py migrate
-```
 
 Import the data into the new database (will overwrite some tables created by Django, but that's ok).
 ```
 cat takwimu/sql/*.sql | psql -U takwimu -W takwimu
 ```
 
+Run migrations to keep Django happy:
+```
+python manage.py migrate
+```
+
```

Comment:
Migrate should always come before loading data from SQL. Let's return this.

Reply:
I think @phillipahereza mention that the other command should start first, 
@phillipahereza what was the reason again for starting with 
```
cat takwimu/sql/*.sql | psql -U takwimu -W takwimu
```
Then 
```
python manage.py migrate
```

Reply:
@mtuchi run `python manage.py migrate` first, it saves you a lot of headache with the loading the geographies mostly

Reply:
@phillipahereza :+1: